doc/conf.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Imports
+
+import sphinx_rtd_theme
+
+# -- Project information -----------------------------------------------------
+
+project = 'galgebra'
+copyright = '2018, Alan Bromborsky and Galgebra team'
+author = 'Alan Bromborsky'
+
+# The short X.Y version
+version = ''
+# The full version, including alpha/beta/rc tags
+release = '0.4.1.1' 
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'nbsphinx',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.viewcode',
+    'IPython.sphinxext.ipython_directive',
+    'IPython.sphinxext.ipython_console_highlighting',
+    'sphinx_markdown_tables',
+]
+
+# -- nbsphinx configuration ---------------------------------------------------
+
+import galgebra
+import nbsphinx
+# nbsphinx_execute = 'always'
+nbsphinx_execute = 'never'
+nbsphinx_allow_errors=True
+nbsphinx_kernel_name='python'
+nbsphinx_timeout = 60
+
+# -- extensions configuration ---------------------------------------------------
+
+napoleon_include_init_with_doc= False
+
+autoclass_content = "both"  # include both class docstring and __init__
+autodoc_default_flags = [
+        # Make sure that any autodoc declarations show the right members
+        "members",
+        "inherited-members",
+        "special-members",
+        "undoc-members",
+        "private-members",
+        "show-inheritance",
```

Comment:
Currently I expose everything in the API doc to make spotting missing docs more easily.